packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,163 @@
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addrA: string
+  let addrB: string
+  let addrCForward: string
+  let addrCCrumbs: string
+  let addrCPie: string
+
+  async function setup (): Promise<void> {
+    addrA = await container.call('getnewaddress')
+    addrB = await container.call('getnewaddress')
+    addrCForward = await container.call('getnewaddress')
+    addrCCrumbs = await container.call('getnewaddress')
+    addrCPie = await container.call('getnewaddress')
+
+    const tokenaddrA = await container.call('getnewaddress')
+    const tokenaddrB = await container.call('getnewaddress')
+    const tokenAddrC = await container.call('getnewaddress')
+    await createToken(tokenaddrA, 'ANT', 100)
+    await createToken(tokenaddrB, 'BAT', 100)
+    await createToken(tokenAddrC, 'CAT', 200)
+
+    await container.call('accounttoaccount', [tokenaddrA, { [addrA]: `${100}@ANT` }])
+    await container.call('accounttoaccount', [tokenaddrB, { [addrB]: `${100}@BAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCForward]: `${49}@CAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCCrumbs]: `${3}@CAT` }])
+    await container.call('accounttoaccount', [tokenAddrC, { [addrCPie]: `${104}@CAT` }])
+    await container.generate(1)
+  }
+
+  async function createToken (address: string, symbol: string, amount: number): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+
+    await container.waitForWalletBalanceGTE(amount)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: `${amount.toString()}@0` }])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+
+  function accToNum (acc: string): number {
+    return Number(acc.split('@')[0])
+  }
+
+  it('should sendTokensToAddress with selectionMode pie', async () => {
+    const accPieBefore = (await client.account.getAccount(addrCPie))[0]
+    const addrReceiver = await client.wallet.getNewAddress()
+    const hex = await client.account.sendTokensToAddress({}, { [addrReceiver]: ['4@CAT'] }, {
+      selectionMode: SelectionModeType.PIE
+    })
+    expect(typeof hex).toStrictEqual('string')
+    expect(hex.length).toStrictEqual(64)
+
+    await container.generate(1)
+
+    const accPieAfter = (await client.account.getAccount(addrCPie))[0]
+    expect(accToNum(accPieAfter)).toStrictEqual(accToNum(accPieBefore) - 4)
+
+    const accReceiver = await client.account.getAccount(addrReceiver)
+    expect(accReceiver[0]).toStrictEqual('4.00000000@CAT')
```

Comment:
The way you write the code is consistent. (See line 84)
`const accReceiver = (await client.account.getAccount(addrReceiver)[0])    expect(accReceiver).toStrictEqual('4.00000000@CAT')`

Reply:
Also, apply to all similar codes in this PR.

Reply:
There is a difference between line 84 and 87. The former (line 84) is an array of balances and later (line 87) is a string.

Reply:
okay

Reply:
Resolved now.